paasta_tools/cli/cmds/security_check.py

```diff
@@ -49,4 +49,4 @@ def perform_security_check(args):
     ret_code, output = _run(security_check_command, timeout=300, stream=True)
     paasta_print("{} exited with code {}".format(security_check_command, ret_code))
     # The security-check should not block the pipeline in case of a failure
```

Comment:
Sure. The pipeline will still not block in case the security-check fails, but this is because of the Jenkins configuration.